resources/icons/16px/excavator.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_9017_3)">
+        <path fill-rule="evenodd" clip-rule="evenodd"
+            d="M14.2193 5.99999C14.6782 5.99999 15.0781 6.31229 15.1894 6.75746L15.6894 8.75746C15.8472 9.38861 15.3699 10 14.7193 10H8.00006C7.44778 10 7.00006 9.55228 7.00006 9V5.30277C7.00006 5.10535 7.0585 4.91234 7.16801 4.74807L8.7032 2.4453C8.88866 2.1671 9.20089 2 9.53525 2H12.0001C12.5523 2 13.0001 2.44771 13.0001 3V5.99999H14.2193ZM10.33 3.99999L9 5.99999L11.0001 5.99999V4L10.33 3.99999Z"
+            fill="#5F6B7C" />
+        <path
+            d="M1.72215 1.11142C1.86912 0.523524 2.50104 0.200416 3.06368 0.425474L5.99093 1.59637C6.53413 1.81365 6.77561 2.44879 6.51397 2.97206L6 3.99999L3 3L2 7.99999H5L2.76294 10.1098C2.50871 10.3496 2.14546 10.4452 1.81522 10.331C0.781705 9.97332 0.0455376 9.42395 0.00206846 8.12352C-0.000663091 8.04181 0.0100623 7.95988 0.029893 7.88056L0.250031 7L1.72215 1.11142Z"
+            fill="#5F6B7C" />
+        <path fill-rule="evenodd" clip-rule="evenodd"
+            d="M3 13.5C3 12.1193 4.11929 11 5.5 11H13.5C14.8807 11 16 12.1193 16 13.5C16 14.8807 14.8807 16 13.5 16H5.5C4.11929 16 3 14.8807 3 13.5ZM5 13.5C5 13.2239 5.22386 13 5.5 13H13.5C13.7761 13 14 13.2239 14 13.5C14 13.7761 13.7761 14 13.5 14H5.5C5.22386 14 5 13.7761 5 13.5Z"
+            fill="#5F6B7C" />
+    </g>
+    <defs>
+        <clipPath id="clip0_9017_3">
+            <rect width="16" height="16" fill="white" />
+        </clipPath>
+    </defs>
```

Comment:
will remove clipPath

![Screenshot 2024-06-28 at 10 53 40 AM](https://github.com/palantir/blueprint/assets/14102129/2d8debef-b304-4b02-86f2-70d23cb026e9)

similar to https://github.com/palantir/blueprint/pull/6705#discussion_r1487982830